lighthouse-core/gather/gatherers/seo/hreflang.js

```diff
@@ -22,7 +22,10 @@ class Hreflang extends Gatherer {
       ).then(attributeValues => attributeValues &&
         attributeValues.map(values => {
           const [href, hreflang] = values;
-          return {href, hreflang};
+          return {
+            href: href || '',
+            hreflang: hreflang || '',
```

Comment:
@kdzwinel the compiler complained that these could be null, not just strings. It is correct wrt `getAttribute`, and I guess there could be some malformed <link>s in pages? However the audit was expecting only strings here, so I'm not sure if I'm missing something or we just aren't covering that possibility yet

Reply:
It looks like you found a bug (this whole typescript thing might be a good idea after all! 😉) . Hreflang audit is certainly not prepared for hreflang being null (https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/audits/seo/hreflang.js#L34).

Reply:
are you ok with leaving with these defaults for now and you could update to what makes sense when you get the chance? (I assume handle the null case if matching the pattern of the other seo artifacts)

Reply:
> are you ok with leaving with these defaults for now
yeah, that SG